### PR TITLE
Fix copyArtifactory failure

### PR DESCRIPTION
**Pull request by sophia-guo**
Archive tap files for all buildtype
Do not fail the copyArtifact step if no appropriate build is found

Signed-off-by: Sophia Guo <sophia.gwf@gmail.com>

### CODE REVIEW

**Review by andrew-m-leonard — APPROVED** [review]
lgtm

File: pipelines/build/common/openjdk_build_pipeline.groovy [hunk 1]
```diff
@@ -423,30 +423,28 @@ class Build {
...
+                                context.echo " ${jobName} result is ${result}"
+                                if (testJob.getResult() == 'SUCCESS' || testJob.getResult() == 'UNSTABLE') {
+                                    context.sh "rm -f workspace/target/AQATestTaps/*.tap"
+                                    try {
+                                        context.timeout(time: 2, unit: 'HOURS') {
+                                            context.copyArtifacts(
+                                                projectName:jobName,
+                                                selector:context.specific("${testJob.getNumber()}"),
+                                                filter: "**/${jobName}*.tap",
+                                                target: "workspace/target/AQATestTaps/",
```
Comment (smlambert) [inline pipelines/build/common/openjdk_build_pipeline.groovy:437]: Any chance I can convince you to call the folder AQAvitTaps ... 3Q is all about branding...

**Review by smlambert — COMMENTED** [review]
thanks for the quick fix, one minor request of naming the archive folder ...

File: pipelines/build/common/openjdk_build_pipeline.groovy (hunk 1 above)
Comment (sophia-guo) [reply pipelines/build/common/openjdk_build_pipeline.groovy]: Updated. Thanks @smlambert

**Review by smlambert — APPROVED** [review]
thanks @sophia-guo !

**Pull request merged**